tethys_cli/db_commands.py

```diff
@@ -0,0 +1,144 @@
+                           help="Password for super user.")
+    db_parser.add_argument('-N', '--superuser-name', dest='superuser_name',
+                           help="Name of super user to add to database when creating.")
+    db_parser.add_argument('-P', '--superuser-password', dest='superuser_password',
+                           help="Password for super user.")
+    db_parser.add_argument('--portal-superuser-name', '--pn', dest='portal_superuser_name',
+                           help="Password for super user.")
+    db_parser.add_argument('--portal-superuser-email', '--email', '--pe', dest='portal_superuser_email',
+                           help="Name of super user to add to database when creating.")
+    db_parser.add_argument('--portal-superuser-password', '--pp', dest='portal_superuser_password',
+                           help="Password for super user.")
+    db_parser.set_defaults(func=db_command, db_alias='default', username='tethys_default', password='pass',
+                           superuser_name='tethys_super', superuser_password='pass',
+                           portal_superuser_name='admin', portal_superuser_email='', portal_superuser_password='pass')
+
+
+def init_db_server(db_dir=None, **kwargs):
+    args = ['initdb', '-U', 'postgres', '-D', f'{db_dir}/data']
+    run_process(args)
+
+
+def start_db_server(db_dir=None, port=None, **kwargs):
+    args = ['pg_ctl', '-U', 'postgres', '-D', f'{db_dir}/data', '-l', f'{db_dir}/logfile', 'start', '-o', f'-p {port}']
+    run_process(args)
+
+
+def stop_db_server(db_dir=None, **kwargs):
+    args = ['pg_ctl', '-U', 'postgres', '-D', f'{db_dir}/data', 'stop']
+    run_process(args)
+
+
+def create_db_user(port=None, username=None, password=None, db_name=None, is_superuser=False, **kwargs):
+    db_name = db_name or username
+
+    if is_superuser:
+        command = f"CREATE USER {username} WITH CREATEDB NOCREATEROLE SUPERUSER PASSWORD '{password}';"
+    else:
+        command = f"CREATE USER {username} WITH NOCREATEDB NOCREATEROLE NOSUPERUSER PASSWORD '{password}';"
+
+    args = ['psql', '-U', 'postgres', '-p', f'{port}', '--command', command]
+    run_process(args)
+    if not is_superuser:
+        args = ['createdb', '-U', 'postgres', '-p', f'{port}', '-O', username, db_name, '-E', 'utf-8', '-T',
+                'template0']
+        run_process(args)
+
+
+def create_tethys_db(port=None, db_name=None, username=None, password=None,
+                     superuser_name=None, superuser_password=None, **kwargs):
+    create_db_user(port=port, username=username, password=password, db_name=db_name)
+    if superuser_name is not None and superuser_password is not None:
+        create_db_user(port=port, username=superuser_name, password=superuser_password, is_superuser=True)
+
+
+def sync_tethys_db(db_alias=None, **kwargs):
+    manage_path = get_manage_path(None)
+    db_alias = db_alias or 'default'
+    args = ['python', manage_path, 'makemigrations']
```

Comment:
Take out `makemigrations`. Now that we are managing this, we shouldn't have users make migrations.

Reply:
done